### PR TITLE
An Initial Example of the Automation

### PR DESCRIPTION
**Pull request by sadikassistivlabs**
Not too much else to say that's not in one of the .md files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+- A username
+- A password
+- A hostname (i.e. a public ip address for the machine)
+
+Once you have that, connect to the machine using your RDP client.
+
+For the Microsoft Remote Desktop app, this works as follows
+
+1. Click the plus icon button in the upper left
```
Comment (WestonThayer) [inline README.md:28]: If you want to streamline these, creating an `.rdp` file is super simple:

```
full address:s:34.171.147.192
username:s:weston

```

I'm not aware of how to supply the password in there as well, looked into it once and nothing worked.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+Instead, their connections have to be wrapped by an SSH connection.
+
+### Forwarding to the Playwright Server's Port
+
+To setup the Playwright websocket server's wrapped connection, follow these steps:
+
+1. Copy the following command to a text file and substitute in the username and hostname you received for `username` and `hostname` respectively.
+
+```bash
+ssh -L 127.0.0.1:4284:127.0.0.1:4284 -N username@hostname -v
```
Comment (WestonThayer) [inline README.md]: FYI you can pass multiple `-L` flags, https://github.com/AssistivLabs/AssistivLabs/blob/main/device-mgmt/device-helpers/macos/startup.sh#L40

**Review by WestonThayer — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sadikassistivlabs) [reply README.md:28]: It seems like approximately the same number of clicks/interactions from what I could tell (including copy-pasting from wherever they're going to be getting it from)

Maybe just to keep the flow in one place I'll leave the app-only instructions in

File: README.md (hunk 2 above)
Comment (sadikassistivlabs) [reply README.md]: Interesting! Much simpler

**Pull request merged**